util/log/clog.go

```diff
@@ -37,6 +37,7 @@ import (
 	"time"
 
 	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/util"
```

Comment:
ok, that's when it stops being in PR scope. Can we still not add new dependencies?